### PR TITLE
#23 tag mechnism

### PR DESCRIPTION
**Pull request by Pawelooo**

### CODE REVIEW

File: src/model/config.py [hunk 1]
```diff
@@ -9,7 +9,9 @@
 FILE_TODO_NAME = 'db_todo' + FILE_FORMAT
 FILE_VIDEO_NAME = 'db_video' + FILE_FORMAT
 FILE_MODEL = 'db_model' + FILE_FORMAT
+FILE_TAG = 'db_tag' + FILE_FORMAT
 FILE_LOCATION = '../../resources/data/'
+FILE_LOCATION_TAG = '../../../resources/data/'
```
Comment (JaRutkowski) [inline src/model/config.py:14]: Is the file stored in a diff location, why cannot we use `FILE_LOCATION`? 🤔

File: src/service/tags/tag.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import json
+
+from src.model.config import FILE_ENCODING, FILE_LOCATION_TAG
+
+
+class Tag:
+    def __init__(self, file_name: str):
+        self.file_name = file_name
+        self.values = []  # the db should be updated accordingly
```
Comment (JaRutkowski) [inline src/service/tags/tag.py:9]: Please 🗑️ the comment.

Comment (JaRutkowski) [reply src/service/tags/tag.py:9]: The same with other places.

**Review by JaRutkowski — CHANGES_REQUESTED** [review]

File: src/model/config.py (hunk 1 above)
Comment (Pawelooo) [reply src/model/config.py:14]: Already discuss

**Review by JaRutkowski — CHANGES_REQUESTED** [review]
Please resolve conflicts. ❌

**Review by JaRutkowski — APPROVED** [review]

**Pull request merged**